SETUP/dump_db_schema.template

```diff
@@ -5,7 +5,7 @@
 db_user='<<DB_USER>>'
 db_password='<<DB_PASSWORD>>'
 db_name='<<DB_NAME>>'
-code_dir=<<CODE_DIR>>
+code_dir='<<CODE_DIR>>'
 
 setup_dir=`dirname $0`
```

Comment:
A minor thing, but backticks for expansion is regarded as not-great style, mainly because you can't nest them. Could you make this `setup_dir=$(dirname $0)`?

Reply:
I think something got crossed somewhere because that's what the current version does.

Reply:
I'm talking about the backticks in ``` setup_dir=`dirname $0` ```, but no matter, it's not worth worrying about.

Reply:
Right, but [none of the current code in the PR](https://github.com/DistributedProofreaders/dproofreaders/pull/1128/files) has that?